apps/kaizen/models.py

```
@@ -1,3 +1,56 @@
+    description = models.TextField(blank=True, null=True)
+    priority = models.IntegerField()
+    due_date = models.DateTimeField(blank=True, null=True)
+    completed = models.BooleanField(default=False)
+    completed_time = models.DateTimeField(blank=True, null=True)
+    is_all_day = models.BooleanField(default=False)
+    reminders = models.JSONField(blank=True, null=True)
+    repeat_flag = models.CharField(max_length=50, blank=True, null=True)
+    time_zone = models.CharField(max_length=50, blank=True, null=True)
+    is_highlight = models.BooleanField(default=False)
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)